playground/package.json

```
@@ -8,11 +8,16 @@
     "gatsby"
   ],
   "scripts": {
-    "develop": "gatsby develop",
-    "start": "gatsby develop",
+    "develop": "gatsby develop --open",
+    "start": "gatsby develop --open",
```

```suggestion
    "start": "gatsby develop",
```

Could be used for cypress and thus `open` should not be added.